www/complete/index.html

```diff
@@ -0,0 +1,164 @@
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css">
+
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"/>
+
+    <title>WaziUp</title>
+  </head>
+  <body>
+
+    <!-- Sidebar -->
+    <ul class="sidebar"  id="mtabs">
+        <img src="waziupLogo.png" class="logo" alt="Waziup">
+        <li class="active"><a href="#context" rel="context">Sensor Context</a></li>
+        <li><a href="#value" rel="value">Sensor Value</a></li>
+        <li><a href="#map" rel="map">Map</a></li>
+        <li><a href="#graph" rel="graph">Graph</a></li>  
+    </ul>
+
+    <!-- Page Content -->
+    <div class="content">
+      <div id="mtabs_content_container">
+        <div id="context" class="mtab_content" style="display: block;">
+          <div> 
+              <ul class="list-group">
+                <li class="list-group-item active">Sensor Context</li>
+                <li class="list-group-item" id="sensor_name_context">Name</li>
+                <li class="list-group-item" id="sensor_owner">Owner</li>
+                <li class="list-group-item" id="sensor_domain">Domain</li>
+                <li class="list-group-item" id="sensor_gateway">Gateway</li>
+              </ul>
+          </div>
+        </div>
+        <div id="value" class="mtab_content">
+            <div> 
+                <ul class="list-group">
+                  <li class="list-group-item active">Sensor Value for Measurement <span id="sensor_value_id"></span></li>
+                  <li class="list-group-item" id="sensor_name_value">Name</li>
+                  <li class="list-group-item" id="sensor_last_value">Last Value</li>
+                  <li class="list-group-item" id="sensor_quantity_kind">Quantity Kind</li>
+                  <li class="list-group-item" id="sensor_sensing_device">Sensing Device</li>
+                  <li class="list-group-item" id="sensor_unit">Unit</li>
+                </ul>
+            </div>
+        </div>
+        <div id="map" class="mtab_content">
+            <div id="mapid" style="height:600px"></div>
+        </div>
+        <div id="graph" class="mtab_content">
+            <canvas id="myChart" width="800" height="600">Loading...</canvas>
+        </div>
+      </div>
+    </div>
+
+
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" ></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" ></script>
+    <script src="https://cdn.jsdelivr.net/npm/waziup-js@0.0.18/lib/Waziup.js"></script>
```

Comment:
Could you update to the new version of the library, V1.1.4? In this version, the domain is removed (see http://www.waziup.io/documentation/api/migration_guide_v1.1/)

Reply:
I couldn't find Waziup.js for versions > V1.1.1

Reply:
It is now version V1.1.5: https://www.npmjs.com/package/waziup-js
Minor versions should be compatible, so you can use 1.1.*

Reply:
What I meant is I couldn't find the CDN of the library. There is no https://cdn.jsdelivr.net/npm/waziup-js@1.1.5/lib/Waziup.js for versions V1.1.2 - V1.1.5. While for version V1.1.1 and below there is one https://cdn.jsdelivr.net/npm/waziup-js@1.1.1/lib/Waziup.js .